Assets/Scripts/Dungeon/RoomTemplates.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
```

Comment:
what does this file do, created variables that dont have functionality? If they are being used in another file we should find a way to keep these variables in that file and delete these ones.